client/state/stats/lists/utils.js

```diff
@@ -238,6 +251,55 @@ export const normalizers = {
 	},
 
 	/**
+	 * Returns a normalized statsTopAuthors array, ready for use in stats-module
+	 *
+	 * @param  {Object} data   Stats data
+	 * @param  {Object} query  Stats query
+	 * @param  {Int}    siteId Site ID
+	 * @param  {Object} site   Site Object
+	 * @return {Array}       Normalized stats data
+	 */
+	statsTopAuthors( data, query = {}, siteId, site ) {
+		if ( ! data || ! query.period || ! query.date ) {
+			return [];
+		}
+		const { startOf } = rangeOfPeriod( query.period, query.date );
+		const authorsData = get( data, [ 'days', startOf, 'authors' ], [] );
+
+		return authorsData.map( ( item ) => {
+			const record = {
+				label: item.name,
+				iconClassName: 'avatar-user',
+				icon: parseAvatar( item.avatar ),
+				children: null,
+				value: item.views,
+				className: 'module-content-list-item-large',
+				actions: [ {
+					type: 'follow',
+					data: item.follow_data ? item.follow_data.params : false
+				} ]
+			};
+
+			if ( item.posts && item.posts.length > 0 ) {
+				record.children = item.posts.map( ( child ) => {
+					return {
+						label: child.title,
+						value: child.views,
+						page: site ? '/stats/post/' + child.id + '/' + site.slug : null,
```

Comment:
In the old stats-list we had just passed the slug as a query option to get around this.  I honestly hadn't thought of grabbing the site object from the state tree.  The only potential issue could be the site object not being in the state tree - though highly unlikely in the context of stats :)

I suppose having it in the query object might be faster since it eliminates grabbing the site object from state.  Either way works for me.